src/main/java/uk/gov/justice/hmpps/datacompliance/repository/jpa/model/OffenderDeletionReferral.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.justice.hmpps.datacompliance.repository.jpa.model;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.Length;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Getter
+@Builder
+@NoArgsConstructor
+@EqualsAndHashCode(of = {"referralId"})
+@Table(name = "OFFENDER_DELETION_REFERRAL")
```

Comment:
Not sure if there are any rules / preferences here at MoJ but tables are generally in plural...

Reply:
I think the pluralised table names are really only in NOMIS? I would say outside of that, singular table names seem to be the norm.  Singular table names often make a lot more sense since they avoid English language pluralisation ambiguity so I've decided in this project to make everything singular.